src/main/groovy/io/cify/framework/core/CifyExpectedConditions.groovy

```diff
@@ -0,0 +1,42 @@
+package io.cify.framework.core
+
+import org.openqa.selenium.Point
+import org.openqa.selenium.StaleElementReferenceException
+import org.openqa.selenium.WebDriver
+import org.openqa.selenium.WebElement
+import org.openqa.selenium.support.ui.ExpectedCondition
+
+class CifyExpectedConditions {
+    static Point lastLocation = new Point(0, 0)
+
+    /**
+     * An expectation for checking an element is not moving. Main use case - check for animation end
+     *
+     * @param element the WebElement
+     * @return Instance of ExpectedCondition<Boolean>
+     */
+    static ExpectedCondition<Boolean> elementToBeStable(final WebElement element) {
+        return new ExpectedCondition<Boolean>() {
+            @Override
+            Boolean apply(WebDriver driver) {
+                Point currentLocation = element.location
+                try {
+                    if (currentLocation.x == lastLocation.x && currentLocation.y == lastLocation.y) {
+                        lastLocation = new Point(0, 0)
+                        return true
+                    }
+                    lastLocation.x = currentLocation.x
+                    lastLocation.y = currentLocation.y
+                    return false
+                } catch (StaleElementReferenceException ignored) {
```

Comment:
If another expected occurs - what will be returned from the method?

Reply:
other exceptions will not be caught and the app would crash

Reply:
It might crash in some other place. What might raise an exception here?

Reply:
unknown issue with unknown exception

Reply:
I don't see any other exceptions to be thrown here...